projects/GEDKeeper2/GKUI/BaseWin.cs

```diff
@@ -1376,55 +1455,119 @@ public bool RecordDelete(GEDCOMRecord record, bool confirm)
             {
                 //string xref = record.XRef;
                 string msg = "";
-
                 switch (record.RecordType)
                 {
                     case GEDCOMRecordType.rtIndividual:
-                        msg = string.Format(LangMan.LS(LSID.LSID_PersonDeleteQuery), ((GEDCOMIndividualRecord)record).GetNameString(true, false));
+                    {
+                        msg = string.Format(LangMan.LS(
+                            LSID.LSID_PersonDeleteQuery),
+                            ((GEDCOMIndividualRecord)record).GetNameString(
+                                true, false));

```

Comment:
You shouldn't 😆  You set the rules as owner of this repo, and that's OK! And I have to obey those rules! 👍